src/main/java/com/kuljava/swiatwsi/world/VillageController.java

```diff
@@ -0,0 +1,42 @@
+package com.kuljava.swiatwsi.world;
+
+import com.kuljava.swiatwsi.exceptions.VillageWithNameAlreadyExistsException;
+import com.kuljava.swiatwsi.exceptions.VillagesAmountExceededException;
+import com.kuljava.swiatwsi.services.VillageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/villages")
+public class VillageController {
+
+  private VillageService villageService;
+
+  @Autowired
+  public VillageController(VillageService villageService) {
+    this.villageService = villageService;
+  }
+
+  @GetMapping("/all")
+  public List<Village> getAll() {
+    return villageService.findAllVillages();
+  }
+
+  @PostMapping("/add")
```

Comment:
Maybe u dont need to create two different endpoints? Try POST and GET to /villages